build_docker.sh

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/env bash
+
+## $ BRANCH=`git rev-parse --symbolic-full-name --abbrev-ref HEAD` &&  sudo bash build_docker.sh -e `git rev-parse ${BRANCH}` -s -r -d /home/BROAD.MIT.EDU/lichtens/gatk_docker_staging/
```

Comment:
Either remove this comment or add another commend line that indicates why it is useful to keep it around.